packages/frontend/src/components/RelayClientTestPage.tsx

```diff
@@ -82,7 +82,7 @@ const RelayClientTestPage: React.FC<RelayClientTestPageProps> = ({
         setConnectionStatus("disconnected");
       } catch (err: unknown) {
         console.error("disconnect error", err);
-        setError(err instanceof Error ? err.message : String(err));
+        setErrorMsg(err instanceof Error ? err.message : String(err));
```

Comment:
this one just gave us trouble in the sentry logger.. _some_ Errors don't like to be identified this way it seems...

cc @wanderer